### DISCUSSION TITLE
bad streaming quality

### DISCUSSION THREAD

**Discussion by SekiroRong** [idiap/coqui-ai-TTS#136] [Q&A] [answered]
Thank you for you excellent project.
However, when I try to stream inference the xtts_v2 model according to the tutorial, I only got good audio at the beginning and bad audio at the end especially when inputs are long tedious chinese texts. Short texts works good BTW.
Why the streaming version of the model act like this, and how can I solve the problem to get long and high quality speech.

**Comment by eginhard** [comment, marked as answer]
You need to split the input into sentences, the model is not trained on long texts.
